federation/pkg/federation-controller/sync/controller.go

```diff
@@ -115,7 +115,7 @@ func newFederationSyncController(client federationclientset.Interface, adapter f
 		adapter:               adapter,
 	}
 
-	// Build delivereres for triggering reconciliations.
+	// Build deliverers for triggering reconciliations.
```

Comment:
Please separate these unrelated and non-functional changes into a separate commit.